src/components/forms/RadioGroup/Radio.tsx

```diff
@@ -197,6 +201,7 @@ function Radio(props: CubeRadioProps, ref) {
         hovered: isHovered,
         button: isButton,
       }}
+      data-type={type || 'radio'}
```

Comment:
I suggest setting default values where you're destructing props instead of putting them in place of use.